src/Servers.jl

```diff
@@ -268,250 +449,62 @@ e.g.
         startwrite(http)
         write(http, "response body")
         write(http, "more response body")
+        return
     end
 
     HTTP.listen() do request::HTTP.Request
         @show HTTP.header(request, "Content-Type")
-        @show HTTP.payload(request)
+        @show HTTP.payload(request, String)
         return HTTP.Response(404)
     end
 ```
 """
-listen(f, host::IPAddr=Sockets.localhost, port::Integer=8081; kw...) = listen(f, Sockets.InetAddr(host, port); kw...)
-listen(f, host::AbstractString, port::Integer; kw...) = listen(f, parse(IPAddr, host), port; kw...)
-listen(f, host::AbstractString; kw...) = listen(f, string(host); kw...)
-
-function listen(f::Function,
-                host::Union{Sockets.InetAddr, String},
-                ;
-                ssl::Bool=false,
-                require_ssl_verification::Bool=true,
-                sslconfig::Union{SSLConfig, Nothing}=nothing,
-                pipeline_limit::Int=ConnectionPool.default_pipeline_limit,
-                tcpisvalid::Function=(tcp; kw...)->true,
-                tcpref::Ref=Ref{Base.IOServer}(),
-                reuseaddr::Bool=false,
-                connectioncounter::Base.RefValue{Int}=Ref(0),
-                kw...)
-
-    if ssl && sslconfig === nothing
-        sslconfig = SSLConfig(require_ssl_verification)
-    end
-
-    @info "Listening on: $host"
-    if isassigned(tcpref)
+function listen end
```

Comment:
The order of the core functions was previously `[listen, listenloop, handle_connection, handle_transaction, handle_stream, handle_request]`.
It now looks to be reversed `[handle(h::Handler, t::Transaction), handle(h::Handler, c::Connection), listenloop, listen]` (and there are some utitilty functions like `getinet` and `getsslcontext` interspersed). 
Would you mind reverting the function so that the diffs line-up as much as possible? It is hard to see what has changed.

Reply:
Switched the order back.

Reply:
Thanks, that helps a bit.
It still looks like getsslcontext has moved and the listenloop diff is corrupted by Server2 stuff; and
check_readtimeout being before handle_connection munges the handle_connection diff.

Reply:
The function signature diffs are all corrupted by the change in wrapping style. Can you revert these stylistic changes so that the semantic changes are clear in the diffs?

Reply:
Sorry, at some point a diff is just going to be a diff. I'm happy to help clarify any individual parts that are confusing or unclear. This also isn't that much code; there are 5 main functions in the Servers.jl file with several one-liner "dispatch" methods.

If we want strict standards for the entire repo on wrapping styles & line lengths, then let's open a separate issue and have everyone discuss what would work best for everyone and we can wholesale convert the repo and then strictly require things in future PRs while pointing people to a documented style.